src/cloud-api-adaptor/podvm/qcow2/misc-settings.sh

```diff
@@ -44,8 +44,8 @@ if [[ "$CLOUD_PROVIDER" == "azure" || "$CLOUD_PROVIDER" == "generic" ]] && [[ "$
     sudo apt-get install -y --no-install-recommends libtss2-tctildr0 libtdx-attest
 fi
 
-# Setup oneshot systemd service for AWS and Azure to enable NAT rules
-if [ "$CLOUD_PROVIDER" == "azure" ] || [ "$CLOUD_PROVIDER" == "aws" ] || [ "$CLOUD_PROVIDER" == "generic" ]
+# Install iptables on all providers except docker/vsphere.
+if [ "$CLOUD_PROVIDER" != "vsphere" ] || [ "$CLOUD_PROVIDER" != "docker" ]
```

Comment:
I think this needs to be an and here?
```suggestion
if [ "$CLOUD_PROVIDER" != "vsphere" ] && [ "$CLOUD_PROVIDER" != "docker" ]
```
This might also have been me giving poor guidance as an example. I think this rule is okay, but it would be good to hear from others

Reply:
oops, apologies idea was to give `==` rather than `!=`, let me get it fixed.. thanks for the suggestion.